jupyter-notebooks/analytics/user-guide/1. Getting Started with the Planet Analytics API.ipynb

```diff
@@ -45,16 +61,16 @@
     "This installment of the tutorial series will focus on connecting to the API and serve as an overview of the concepts of **Feeds** and **Subscriptions**, the two basic building blocks of an \"analytics feed\".\n",
     "\n",
     "### Feeds \n",
-    "A **Feed** represents an analytic derived from Planet imagery. Each feed essentially represents an analytic capability that has been uniquely configured to optimize performance and quality, and each have unique types of outputs. For example, a Road Detection Feed represents roads detected on monthly Planet Basemaps, and outputs raster \"segmentation mask\" data. Several types of Feeds are currenlty available on the Planet Analytics API, and we plan on releasing new feeds in the future.\n",
+    "A **Feed** represents an analytic derived from Planet imagery. Each feed essentially represents an analytic capability that has been uniquely configured to optimize performance and quality, and each have unique types of outputs. For example, a Road Detection Feed represents roads detected on monthly Planet Basemaps, and outputs raster \"segmentation mask\" data. Several types of Feeds are currently available on the Planet Analytics API, and we plan on releasing new feeds in the future.\n",
     "\n",
     "### Subscriptions \n",
-    "Users have *subscriptions to feeds* in a specific Area of Interest (AOI) and Time Interval of Interest (TOI). For example, a **Subscription** could be Road Detection over 12 months in San Francisco, California. \"Subscribing\" to a Feed is how a user can initiate the process of leveraging an analytic capability over an AOI/TOI and generate analytics datasets called \"Results\".\n",
+    "Users have *subscriptions to feeds* in a specific Area of Interest (AOI) and Time Interval of Interest (TOI). For example, a **Subscription** could be Road Detections over 12 months in San Francisco, California. \"Subscribing\" to a Feed is how a user can initiate the process of leveraging an analytic capability over an AOI/TOI and generate analytics datasets called \"Results\".\n",
     "\n",
     "We'll be covering how to access the available Feeds and Subscriptions in this tutorial.\n",
     "\n",
     "### Results\n",
     "\n",
-    "When new Planet imagery is published that intersects a Subscription's AOI and TOI, Planet’s computer vision models process the imagery and the output is added to a \"collection\" (WFS 3) of **Results** associated with the Subscription. \n",
```

Comment:
Why are you removing the WFS3 note here?

Reply:
hmm, that looks like a mistake; I'll take a look.

Reply:
I thought the sentence had been removed, but it is still present:

```
    "When new Planet imagery is published that intersects a Subscription's AOI and TOI, Planet’s computer vision models process the imagery and the output is added to a \"collection\" of **Results** associated with the Subscription. \n",
```

I removed the WFS3 note here because this is an introductory sentence, and it didn't seem to make sense to dive into the WFS3 specifics here; those are expanded below in the WFS3 section and callout below.